packages/client/src/v2-events/features/events/AdvancedSearch/utils.ts

```diff
@@ -115,6 +115,13 @@ const RegStatus = {
   Created: 'CREATED'
 } as const
 
+const MatchType = {
+  fuzzy: 'fuzzy',
+  exact: 'exact',
+  anyOf: 'anyOf',
+  range: 'range'
+} as const
```

Comment:
Would it make sense to use the `MatchType` in `AdvancedSearchConfig.ts` instead of creating a new enumeration?

`const MatchType = z.enum(['fuzzy', 'exact', 'range'])`

Reply:
currently, there is no implementation of anyOf. I wonder, that anyOf would support multiple value and need special handling, but here we are defining the baseField which at a time represent a single field value.